Automattic/wp-calypso
# Unpin dependencies (Part 3)

#### Changes proposed in this Pull Request

Third round of unpinning dependencies.

Previous PRs:
* Part 1: #41191
* Part 2: #41472

#### Testing instructions

* Verify there are no packages removed/added in yarn.lock

* Validate the dependency tree has not changed:
    * Install dependencies and generate the tree for this branch: `find . -type d -name node_modules | xargs rm -fr && yarn && NODE_OPTIONS=--max_old_space_size=8192 yarn run effective-module-tree -o list > ../branch-tree`
    * Install dependencies and generate the tree for master (tested with 948b416a34): `find . -type d -name node_modules | xargs rm -fr && yarn && NODE_OPTIONS=--max_old_space_size=8192 yarn run effective-module-tree -o list > ../master-tree`
    * Diff both trees and check there are no differences: `diff ../branch-tree ../master-tree`
